questions/qa-choosing-language-tags.en.html

```diff
@@ -331,11 +331,16 @@ <h3>Decision 6: Extension subtags</h3>
     </ul>
     <p>The <code class="kw" translate="no">u-</code> extension is defined in <a href="https://www.rfc-editor.org/rfc/rfc6067">RFC 6067</a>, which points to the Unicode Consortium's Common Locale Data Repository (CLDR) for details on the subtags that follow it. It is not defined by BCP 47.</p>
     
-    <p>The subtag <code class="kw" translate="no">t</code> was also registered by the Unicode Consortium and is used for transformed content. It allows for the specification of content that has been transliterated, transcribed, or translated, or in some other way influenced by the source language. For example, the following indicates Japanese content that has been transformed from Italian:</p>
+    <p>The subtag <code class="kw" translate="no">t</code> was also registered by the Unicode Consortium and is used for transformed content. It allows for the specification of content that has been transliterated, transcribed, or translated, or in some other way influenced by the source language. For example, the following indicates Ukrainian text that has been transcribed from Cyrillic to Latin:</p>
     <ul>
-      <li><code class="kw" translate="no">ja-t-it</code></li>
+      <li><code class="kw" translate="no">uk-Latn-t-uk-cyrl</code></li>
     </ul>
     <p>The <code class="kw" translate="no">t-</code> extension is defined in <a href="https://www.rfc-editor.org/rfc/rfc6497">RFC 6497</a> and, like the <code class="kw" translate="no">u-</code> extension, the details of the subtags that follow it are provided by the Unicode Consortium's CLDR.</p>
+
+    <p>Additionally, the field separator subtag <code class="kw" translate="no">m0</code> can be used before certain subtags. For example, the following indicates a dated version of the UNGEGN transliteration specification for Hebrew to Latin:</p>
+    <ul>
+      <li><code class="kw" translate="no">he-Hebr-t-he-latn-m0-ungegn-2007</code></li>
```

Comment:
Shouldn't this be the following?
he-Latn-t-he-Hebr-m0-UNGEGN-2007

See the previous example, which labels Ukrainian in Latin script as a transcription of Cyrillic Ukrainian.

Reply:
fixed